### DISCUSSION TITLE
Prevent reviews from being dismissed in a stack

### DISCUSSION THREAD

**Discussion by justinanchor** [Mergifyio/mergify#5124] [Q&A]
I'm enjoying using Mergify Stacks except one detail with reviews being dismissed.

I have a Stack of PRs and each one has been approved by reviewers before any have merged. When the first PR merges, Mergify dismisses the existing review and comments "The base branch has changed". My configuration does not use the dismiss_reviews action.

Is there a way to configure Mergify so reviews are not dismissed when a depends-on stacked PR merges?

**Comment by jd** [comment]
Hi @justinanchor ! happy to see you like it. Mergify does not dismiss review unless you instruct it to do so.

Could you share a screenshot of what you see? It might be a side-effect of GitHub settings.

**Reply by justinanchor** [reply]
Thanks jd. Could this be confusing Github UX because the mergify bot is what updated the base branch?

![Screenshot 2024-09-12 12 24 17 PM](https://github.com/user-attachments/assets/1d46b6fe-cea7-4881-888e-0d11847d785c)

**Reply by jd** [reply]
Did you enable GitHub branch protection settings for dismissing approvals?
